### PR TITLE
Create money

### PR DESCRIPTION
**Pull request by Johnchiahk**

### CODE REVIEW

File: src/main/java/arb/model/project/Price.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public String fullPrice;
+
+    /**
+     * Constructs a {@code Price}.
+     * @param price A valid price.
+     */
+    public Price(String price) {
+        requireNonNull(price);
+        checkArgument(isValidPrice(price), MESSAGE_CONSTRAINTS);
+        fullPrice = price;
```
Comment (zrei) [inline src/main/java/arb/model/project/Price.java:27]: After validating the `price [String]` you can just do `this.price = Double.parseDouble(price);` to parse it to a double and store it that way instead.

Comment (Johnchiahk) [reply src/main/java/arb/model/project/Price.java:27]: noted with thanks

Comment (zrei) [reply src/main/java/arb/model/project/Price.java:27]: Then to turn it to a string to store into storage, you can use `Double.toString()`

**Review by zrei — APPROVED** [review]
LGTM

**Pull request merged**